database/database.go

```diff
@@ -19,40 +24,67 @@ var mkdirAll = os.MkdirAll
 func New(dialect, connection, defaultUser, defaultPass string, strength int, createDefaultUserIfNotExist bool) (*GormDatabase, error) {
 	createDirectoryIfSqlite(dialect, connection)
 
-	db, err := gorm.Open(dialect, connection)
+	logLevel := logger.Info
+	if mode.Get() == mode.Prod {
+		logLevel = logger.Warn
+	}
+
+	dbLogger := logger.New(log.New(os.Stderr, "\r\n", log.LstdFlags), logger.Config{
```

Comment:
Why the windows line endings??

Reply:
I copied it from the official code, didn't ask too many questions:

https://github.com/go-gorm/gorm/blob/b88148363a954f69fa680b152dfd96a94ffea1e1/logger/logger.go#L75